ImageInLib/ImageInLib/src/fast_sweeping_method.c

```diff
@@ -48,13 +53,15 @@ bool fastSweepingFunction_3D(dataType ** distance3DPtr, dataType ** curve3DPtr,
 			for (j = 0; j < yDim; j++)
 			{
 				i_n = i + j * xDim;
-				if (curve3DPtr[k][i_n] == fgroundValue)
+
+				if (curve3DPtr[k][i_n] < threshold + range)
```

Comment:
again just an optimization issue
Please extract threshold + range as const double (do not evaluate it in each loop)

Reply:
Okays